src/core/css/inc/vars.scss

```diff
@@ -2,20 +2,23 @@
 $primary-font-color: #000;
 $body-font-color: #1d1d1d;
 $header-font-color: #fff;
-$header-background: #1f386e;
+// $header-background: #1f386e;
 
 // Colours
 $text-color-default: #333;
 $text-color-headings: $text-color-default;
 $text-color-message: #fff;
 
-$body-color: #fff;
+// This is the color used for the MastHead background and the basis of the
+// SearchForm and Footer background colors.
+$base-background-color: #1e1e1e;
+$body-and-accent-color: #ffb200;
```

Comment:
These seem to be describing how we use them rather than why we use them. The masthead colour seems a little hard to name but maybe there's something good.

Seems like it could be `$body-and-accent-color` ~> `$accent-color`, `$background-accent-color` ~> `$base-color`, `$base-background-color` ~> `$i-give-up-just-put-masthead-in-the-name`.